src/Components/Aspire.Azure.AI.OpenAI/AzureOpenAISettings.cs

```diff
@@ -70,6 +70,11 @@ public bool DisableTracing
         set { _disableTracing = value; }
     }
 
+    /// <summary>
+    /// Gets the models to use for the Azure OpenAI resource.
```

Comment:
Can the comment explain what the keys and values represent? It's not clear from the property signature alone.